Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -915,6 +950,7 @@ void doVayDepositionShapeN (const GetParticlePosition& GetPosition,
         }
 #endif
     } );
+    amrex::The_Managed_Arena()->free(cost_real);
```

Comment:
As we cleaned this up from the original research prototypes (https://doi.org/10.5281/zenodo.4708449), we forgot to add the addition to `*costs` at this point.

Measurements were done with the code linked in the zenodo archive.